client/pom.xml

```diff
@@ -20,6 +20,7 @@
     <properties>
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
+        <assertj.version>2.5.0</assertj.version>
```

Comment:
Seems like this is unrelated and maybe just came along for the ride.  Is it needed?